.github/workflows/build_test_ci.yml

```
@@ -9,9 +9,9 @@ name: CI
 
 on:
   push:
-    branches: [ main, 2025 ]
+    branches: [ main, 2025, 2026 ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)